### PR TITLE
fix: changes required for Kafka SDK update

### PR DESCRIPTION
**Pull request by wtrocki**
I have verified all commands manually:

- Topics
- Consumer-Groups
- ACLs

Just minor changes were required.

### CODE REVIEW

File: pkg/cmd/kafka/consumergroup/resetoffset/reset_offset.go [hunk 1]
```diff
@@ -133,8 +133,11 @@ func runCmd(opts *options) error {
 		return err
 	}
 
+	offsetEnum, _ := kafkainstanceclient.NewOffsetTypeFromValue(opts.offset)
```
Comment (wtrocki) [inline pkg/cmd/kafka/consumergroup/resetoffset/reset_offset.go:136]: Not sure if we should handle error as we are doing validation before so we are guaranteed to have valid offset.

File: pkg/shared/connection/api/defaultapi/default_client.go [hunk 2]
```diff
@@ -154,7 +154,7 @@ func (a *defaultAPI) KafkaAdmin(instanceID string) (*kafkainstanceclient.APIClie
 		apiHost := fmt.Sprintf("admin-server-%v", host)
 		apiURL, _ = url.Parse(apiHost)
 		apiURL.Scheme = "https"
-		apiURL.Path = "/rest"
+		apiURL.Path = "/"
```
Comment (wtrocki) [inline pkg/shared/connection/api/defaultapi/default_client.go:157]: This should have zero impact when we move to /api/v1

File: pkg/cmd/kafka/consumergroup/resetoffset/reset_offset.go (hunk 1 above)
Comment (rkpattnaik780) [reply pkg/cmd/kafka/consumergroup/resetoffset/reset_offset.go:136]: We can skip handling here.

**Review by rkpattnaik780 — APPROVED** [review]

**Comment by wtrocki** [comment]
@rkpattnaik780 I will merge so we can do automatic tests and rebase other PRs

**Pull request merged**